common/autoware_universe_utils/include/autoware/universe_utils/ros/diagnostics_module.hpp

```diff
@@ -22,7 +22,7 @@
 #include <string>
 #include <vector>
 
-namespace autoware::localization_util
+namespace autoware::universe_utils
 {
 class DiagnosticsModule
```

Comment:
IMO DiagnosticInterface is better

Reply:
Fixed: https://github.com/autowarefoundation/autoware.universe/pull/9714/commits/b7a90d82971fa10768ed0d5525df6b7bafb9bccb